multiply_ui/server/resources/processing-parameters.json

```diff
@@ -102,8 +102,8 @@
       "id": "s2_prosail",
       "name": "PROSAIL for Sentinel-2",
       "description": "Coupling of PROSPECT leaf optical properties model and SAIL canopy bidirectional reflectance model. It links the spectral variation of canopy reflectance, which is mainly related to leaf biochemical contents, with its directional variation, which is primarily related to canopy architecture and soil/vegetation contrast.",
-      "modelAuthor": "",
-      "modelurl": "http://teledetection.ipgp.jussieu.fr/prosail/",
+      "modelAuthors": "",
```

Comment:
If it's plural, make it a list.

Reply:
Why?

Reply:
Have a look at the `valueRange` that you defined; it is a string rather than a range *range*.

Reply:
If you make valueRange a range, I'll make modelAutors a list :)